Cargo.toml

```diff
@@ -27,6 +27,9 @@ bitflags = "0.9"
 cfg-if = "0.1.0"
 void = "1.0.2"
 
+[target.'cfg(target_os = "dragonfly")'.dependencies]
+errno-dragonfly = "0.1"
```

Comment:
That's ... sad.  I took a stab at this myself.  It doesn't look as though we can use `#[thread_local]` unless we enable it for _all_ builds, not just Dragonfly's.  Still, what would you say to moving the C extension into Nix instead of making it its own crate?  Cargo just seems like an awfully heavyweight way to manage such a tiny amount of code.